tests/Fixtures/Entity/Embeddable/SubEmbeddedEntity.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Fixtures\Entity\Embeddable;
+
+class SubEmbeddedEntity
+{
+    /**
+     * @var bool
+     */
+    protected $plainField;
+}
```

Comment:
Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.

Reply:
Should be OK